### PR TITLE
Added Sliding Window Technique

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

Given an array of N elements and a value K (K <= N) , calculate maximum summation of K consecutive elements in the array.
Can be solved this problem by using Sliding Window Technique.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2689

### CODE REVIEW

File: C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Given an array of N elements and a value K (K <= N) , calculate maximum summation of K consecutive elements in the array.
+
+/*
+We can solve this problem by using Sliding Window Technique. This technique will allow us to solve this problem in O(N)
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp]: Enclose them in a single comment block

File: C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+5 3
+5 2 -1 0 3
+maximum summation of K consecutive elements in the array is 6
+
+
+
+Time Complexity : O(N)
+Space Complexity : O(1)
+
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp]: Add a new line at the EOF

File: C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        {
+            win_sum += (ar[i] - ar[i - K]);
+            if(win_sum > max_sum)
+            {
+                max_sum = win_sum;
+            }
+        }
+        return max_sum;
+
+
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp]: Remove extra lines of code

File: C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    }
+    if(N > K)
+    {
+        // calculate sum of first window of size K
+        int max_sum = 0, win_sum = 0;
+        for(int i = 0; i < K; i++)
+            max_sum += ar[i];
+
+        //calculate sum of remaining windows
+        // by removing elements from first window simultaneously add elements to current window
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Calculate_max_sum_of_k_consecutive_elements.cpp]: Use multi-line comments

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
I have made all the changes. Please review.

**Comment by RaiyanMahin** [comment]
I don't know why  1 check was failed :(
@HarshCasper

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**